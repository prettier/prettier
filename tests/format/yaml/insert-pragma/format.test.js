runFormatTest(import.meta, ["yaml"], { insertPragma: true });
