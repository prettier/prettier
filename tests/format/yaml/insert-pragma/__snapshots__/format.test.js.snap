// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with-pragma.yml - {"insertPragma":true} format 1`] = `
====================================options=====================================
insertPragma: true
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
# @prettier

    123

=====================================output=====================================
# @prettier

123

================================================================================
`;

exports[`without-pragma.yml - {"insertPragma":true} format 1`] = `
====================================options=====================================
insertPragma: true
parsers: ["yaml"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================


    123

=====================================output=====================================
# @format

123

================================================================================
`;
